src/wagtail_vector_index/backends/qdrant/backend.py

```diff
@@ -3,7 +3,7 @@
 
 from qdrant_client import QdrantClient
 from qdrant_client.http import models as qdrant_models
-from wagail_ai.index import Document
+from wagail_ai.index.base import Document
```

Comment:
Think this needs to be `from wagtail_vector_index.base`

Reply:
gah, not sure how that happened. fixed and rebased